components/Dialogs/src/MDCDialogTransitionController.h

```diff
@@ -34,4 +34,19 @@
 @interface MDCDialogTransitionController
     : NSObject <UIViewControllerAnimatedTransitioning, UIViewControllerTransitioningDelegate>
 
+/**
+ Duration of transition animation.
+
+ Defaults to 0.27.
+ */
+@property(nonatomic, assign) NSTimeInterval animationDuration;
+
+/**
+ The starting scale factor of the dialog, between 0 and 1. The "animate in" transition scales the
+ dialog from this value to 1.0.
+
+ Defaults to 1.0 (no scaling is performed).
+ */
+@property(nonatomic, assign) CGFloat dialogScaleFactor;
```

Comment:
Nit: Naming: Consider making the name more explicit that this only applies to the "animate-in" starting scale.  For example, `presentationAnimationStartingScaleFactor`. A later API could include a `dismissAnimationEndingScaleFactor` for dismissal configuration.